tools/cd_scripts/package_gcsfuse.sh

```diff
@@ -64,6 +64,10 @@ function exit_in_failure() {
 RELEASE_VERSION=$(fetch_meta_data_value "RELEASE_VERSION")
 echo RELEASE_VERSION="$RELEASE_VERSION"
 
+# '~' is not accepted as docker build tag and git tag. Hence, we will it instead of RELEASE_VERSION.
+RELEASE_VERSION_TAG=$(echo $RELEASE_VERSION | tr '~' '_')
+echo RELEASE_VERSION_TAG="$DOCKER_BUILD_TAG_SUFFIX"
```

Comment:
I didn't follow this change. What is DOCKER_BUILD_TAG_SUFFIX?

Reply:
Missed by mistake, removing that.